arches/app/models/models.py

```diff
@@ -725,6 +727,12 @@ class Meta:
             models.UniqueConstraint(fields=["name", "nodegroup"], name="unique_nodename_nodegroup"),
             models.UniqueConstraint(fields=["alias", "graph"], name="unique_alias_graph"),
         ]
+        indexes = [
+            models.Index(
+                Cast(KT("config__controlledList"), output_field=models.UUIDField(null=True)),
```

Comment:
I didn't realize this was available from the ORM. It seems worth looking into other places in Arches where we should be adding JSON indexes.

I did a little test to confirm it's being used (and it is). 
```
EXPLAIN(SELECT n."nodeid" 
FROM "nodes" n 
WHERE ((n."config" ->> 'controlledList'))::uuid = ('06b58ca6-bcb6-46b3-8aaa-64cca94ab8f8'))
```

Reply:
Smart! (One thing I've done recently is to just call `.explain()` on the queryset and log out the result.)

Reply:
> Smart! (One thing I've done recently is to just call `.explain()` on the queryset and log out the result.)

Oh, that's really cool. I think @apeters would definitely appreciate that trick.

Reply:
That is cool!   🥇